app/components/forms/song/LineFields/LineFields.tsx

```diff
@@ -9,17 +9,20 @@ import { Fieldset, useNamespace } from "~/lib/fieldset";
 import { useArray } from "~/lib/useArray";
 import { useFocus } from "~/lib/useFocus";
 import uniqid from "uniqid";
+import { convertPlainTextToLines } from "~/helpers/convertPlainTextToLines";
 
 type Props = {
   line: ILine;
-  insertLine?: () => void;
+  insertLine?: (lyric?: string) => void;
```

Comment:
insertLine now takes an optional `lyrics` param so we can add a new line with lyrics already embedded in it